usr/lib/mate-hud/mate-hud

```diff
@@ -174,21 +190,59 @@ def get_menu(menuKeys):
     height_dpi = get_dpi(screen.height(), screen.height_mm())
     dpi = scale * (width_dpi + height_dpi) / 2
 
-    menu_cmd = subprocess.Popen(['rofi', '-dmenu', '-i',
-                                 '-location', '1',
-                                 '-width', '100', '-p', 'HUD',
-                                 '-lines', '10', '-font', font_name,
-                                 '-dpi', str(dpi),
-                                 '-separator-style', 'none',
-                                 '-hide-scrollbar',
-                                 '-click-to-exit',
-                                 '-levenshtein-sort',
-                                 '-line-padding', '2',
-                                 '-kb-cancel', 'Escape' + shortcut,
-                                 '-sync', # withhold display until menu entries are ready
-                                 '-monitor', '-2', # show in the current application
-                                 '-theme', get_rofi_theme()],
-                                 stdout=subprocess.PIPE, stdin=subprocess.PIPE)
+    theme_options = 'window { location: northwest;'
+
+    rofi_theme = get_rofi_theme()
+    cmd = ['rofi', '-dmenu', '-i',
+           '-p', 'HUD',
+           '-lines', '10', '-font', font_name,
+           '-dpi', str(dpi),
+           '-separator-style', 'none',
+           '-hide-scrollbar',
+           '-click-to-exit',
+           '-levenshtein-sort',
+           '-line-padding', '2',
+           '-kb-cancel', 'Escape' + shortcut,
+           '-sync', # withhold display until menu entries are ready
+           '-monitor', '-2', # show in the current application
+           '-theme', rofi_theme,
+           '-theme-str', 'window { location: north west; ' + \
```

Comment:
Why not provide this as part of the rasi theme itself, and rely on `-location 1` for all other windows?

Reply:
I'm pretty sure the location switch is deprecated [link](https://man.archlinux.org/man/community/rofi/rofi.1.en#Layout_and_Theming) which is why I switched to the new theme format way of doing it.